easybuild/easyblocks/g/gcc.py

```diff
@@ -410,6 +410,10 @@ def build_step(self):
 
             # enable bootstrapping for self-containment
             configopts += " --enable-bootstrap "
+            # Use 'bootstrap-debug' build configuration to force stripping of object
+            # files prior to comparison during bootstrap (broken by Xcode 6.3).
+            # see https://github.com/Homebrew/homebrew/issues/38501
+            configopts += "--with-build-config=bootstrap-debug "
```

Comment:
at best, we should only do this when the OS is detected as being OS X (via `get_os_type()` from `easybuild.tools.systemtools`)

it's not clear to me what other implications this extra configopts would have